resources/providers/awslib/elb_provider.go

```diff
@@ -26,6 +26,7 @@ import (
 )
 
 type ElbLoadBalancerDescriptions []types.LoadBalancerDescription
```

Comment:
nit: isn't the `ElbLoadBalancerDescriptions` an array of `ElbLoadBalancersDescription`?

Do we need them both?